### ISSUE TITLE
Parallel request to the server

### ISSUE THREAD

**Issue by alpcansoydas** [collabora/WhisperLive#188]
For example, the server side is deployed. Can the server-side handle multiple parallel transcription requests? How many requests can it be handled? Will there be any performance issues?

**Comment by cjpais** [comment]
Yes it can. On a RTX4080 I am able to get 4 parallel streams without issue, this is specified in the TranscriptionServer class. It's default is 4, but you can do more. 

Each stream loads into VRAM, so you will be limited by VRAM.
